Index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <link rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/fontawesome.min.css"
+        integrity="sha384-QYIZto+st3yW+o8+5OHfT6S482Zsvz2WfOzpFSXMF9zqeLcFV0/wlZpMtyFcZALm" crossorigin="anonymous">
+</head>
+<body>
+    <header>
+        <span><a href="">My Logo</a></span>
+        <nav id="switch">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </nav>
+    </header>
+    <section class="section1">
+        <h1 id="greetings">I’m Alexandra. Glad to see you!</h1>
+        <p id="details">I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate
+            to contact me.</p>
+    </section>
+    <footer>
+        <span> LET’S CONNECT</span>
+        <ul>
+           <li><a href="#"><img src="./images/twitter.png"></a></li>
+           <li><a href="#"><img src="./images/linkedin.svg"></li>
+           <li><a href="#"><img src="./images/m.png"></a></li>
+           <li><a href="#"><img src="./images/git.png"></a></li>
+           <li><a href="#"><img src="./images/peace.png"></a></li>
+        </ul>
+    </footer>
```

Comment:
**OPTIONAL** - Good job done so far, but to help improve the semantics of your page, I do not advise wrapping this section in a footer tag because it is part of the main content of your page. Consider wrapping it in a `div` tag and placing it in the `section` tag above. i.e it should be a direct child of that section. Please fix this 🙏